src/createWebStoragePersistor-experimental/index.ts

```diff
@@ -48,11 +48,13 @@ function throttle<TArgs extends any[]>(
   wait = 100
 ) {
   let timer: number | null = null
-
+  let params: TArgs
   return function (...args: TArgs) {
-    if (timer === null) {
+    if (timer !== null) {
```

Comment:
I think we can remove the if? If the throttled function is called only once, we will invoke it with undefined after setTimeout

Reply:
yes, it is a mistake.I will fix it.